modules/oe_theme_content_event/templates/oe-theme-content-event-contacts.html.twig

```diff
@@ -13,7 +13,7 @@
       <div id="event-contacts-general">
         <h3 class="ecl-u-type-heading-3 ecl-u-mt-none ecl-u-mb-m ecl-u-mb-m-l">{{ 'General contact'|t }}</h3>
         {% for contact in general %}
-          <div{{ not loop.last ? ' class="ecl-u-mb-xl ecl-u-pb-xl ecl-u-border-bottom ecl-u-border-color-grey-20"'}}>
+          <div{{ not loop.last ? ' class="ecl-u-mb-xl ecl-u-pb-xl ecl-u-border-bottom {{ ecl_class_border_color() }}"'}}>
```

Comment:
We have two prints inside one another ({{ ecl_class_border_color() }} is inside another {{ }}) and it just prints it as a string. The resulting div is: <div class="ecl-u-mb-xl ecl-u-pb-xl ecl-u-border-bottom {{ ecl_class_border_color() }}">

Reply:
I think this is not tested either in tests/src/Functional/ContentEventRenderTest.php so maybe a good time to add an assertion for it.